Source/AliveLibAO/DoorFlame.cpp

```diff
@@ -238,7 +239,47 @@ class FlameSparks : public BaseAnimatedWithPhysicsGameObject
 
     EXPORT void VUpdate_432440()
     {
-        NOT_IMPLEMENTED();
+        PSX_RECT rect = {};
+        gMap_507BA8.Get_Camera_World_Rect_444C30(CameraPos::eCamCurrent_0, &rect);
+        field_A8_xpos = FP_FromInteger(rect.w + 16);
+        field_AC_ypos = FP_FromInteger(rect.y + 0xFFF0);
```

Comment:
is that not -16 or something? 🤔

Reply:
Nope, not sure what that is, because if it was -16, the number would be twice as long (rect.y is short) - in AE it *is* twice as long in IDA...